uberon_edit.obo

```diff
@@ -195227,6 +195227,25 @@ relationship: dc-creator http://orcid.org/0000-0003-2034-601X
 relationship: part_of UBERON:0004647 ! liver lobule
 property_value: http://purl.org/dc/elements/1.1/date 2020-10-05T15:52:22Z xsd:dateTime
 
+[Term]
+id: UBERON:8410011
+name: inguinal region skin
+def: "The skin covering the inguinal region between the abdomen and the thigh on either side of the pubic bone." []
+synonym: "groin skin" EXACT []
+synonym: "inguinal skin" EXACT []
+synonym: "skin of groin" EXACT []
+synonym: "skin of inguinal region" EXACT []
+synonym: "zone of skin of groin" EXACT []
+synonym: "zone of skin of inguinal region" EXACT []
+xref: http://orcid.org/0000-0001-7505-5418
+xref: https://en.wikipedia.org/w/index.php?title=Groin&oldid=998393002
```

Comment:
Should be curie.  Check GO xref abbs for prefix.

Reply:
AI for @dosumis : figure out curie format for Wikipedia revision history (so it's picked up correctly by OLS) and report.
AI for @paolaroncaglia : then fix all Wikipedia links in all my PRs/terms.

Reply:
Please use WikipediaVersioned:Groin&oldid=998393002